### ISSUE TITLE
Error - Connection failure

### ISSUE THREAD

**Issue by m-boyd** [nvllsvm/Audinaut#15]
# Environment

 * QNAP 4.3.3.0136 Web Server
 * Ampache 3.8.2
   - Subsonic backend (Access Level: User)

# Description

Error message after configuring Ampache server with Subsonic backend enabled: "Connection failure. A network error occurred. Please check the server address or try again later."

**Comment by nvllsvm** [comment]
Audinaut will only officially support Libresonic as a server. Other servers may will not be looked into by me, though I may accept a patch if someone makes one.

**Comment by anotherwon** [comment]
AFAIK ampache implements an old API version.
See also: https://github.com/ampache/ampache/issues/1405

**Comment by lachlan-00** [comment]
i've started updating the subsonic api in ampache https://github.com/ampache/ampache/pull/1965

Right now it's just supporting token auth with user + apikey but that's a start!
I'm going to bump it up to 1.13.0 which seems to be the minimum for token auth.

**Comment by lachlan-00** [comment]
I've added the missing methods from 1.13 into my pull.
Just waiting on another pull before i put it in develop as well as check for missing attributes.

What's the minimum subsonic API version Audinaut supports? it seems okay on my server but i can't really tell if it needs anything extra compared to dsub/ultrasonic.

I was just going to stop at 1.13 but if there's a need i can bump it up.

**Comment by lachlan-00** [comment]
My pull will be merged this week once i've done some more testing. Thanks for waiting 2 years. :/

**Comment by lachlan-00** [comment]
@m-boyd token auth is merged into the latest Ampache develop
